.changeset/orange-dancers-compete.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixed: CLI to avoid waiting for stdin without any input (regression since 14.10.0)
```

Comment:
```suggestion
Fixed: CLI regression to avoid waiting for stdin without any input
```

Avoids brackets, as we typically only use them for contributor and pull requests.